src/site/stages/build/drupal/graphql/vaForms.graphql.js

```diff
@@ -0,0 +1,45 @@
+module.exports = `
+fragment vaFormPage on NodeVaForm {
+  title
+  fieldVaFormName
+  fieldVaFormTitle
+  fieldVaFormType
+  fieldVaFormUrl {
+    uri
+    title
+    options
+  }
+  fieldVaFormUsage {
+    value
+    format
+    processed
+  }
+  fieldVaFormNumber
+  fieldAdministration {
+    targetId
+  }
+  fieldVaFormAdministration {
+    targetId
+  }
+  fieldVaFormToolIntro
+  fieldVaFormToolUrl {
+    uri
+    title
+    options
+  }
+  fieldBenefitCategories {
+    targetId
+  }
+  fieldVaFormRevisionDate {
+    value
+    date
+  }
+  fieldVaFormIssueDate {
+    value
+    date
+  }
+  fieldVaFormNumPages
+  status
+  moderationState
+}
+`;
```

Comment:
Copied from https://dsva.slack.com/archives/CUB5X5MGF/p1592940541321700 and tested on https://prod.cms.va.gov/graphql/explorer